devops/setup-gpg/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: 'Setup GPG'
+description: 'Configures this action to run gpg with a given key and pass'
```

Comment:
It might help to specify which OS'es are supported. Github supports composite actions for all their OS'es (i.e Windows, Linux, macOS)

Reply:
good point. linux is the only thing I know it works on.

Reply:
Not sure why we need interactive mode, because that would exposed private key and passphrase.

Reply:
I dont know what you mean, would you like to put this comment somewhere else?

Reply:
In the action description you can add that this composite action is only supported on Linux